service/telemetry.go

```diff
@@ -31,76 +20,26 @@ const (
 
```

Comment:
nit: zapKeyTelemetryAddress and zapKeyTelemetryLevel are not used anymore, could be removed